cmd/ecsReplaceInstances.go

```diff
@@ -30,13 +31,11 @@ var replaceInstancesCmd = &cobra.Command{
 	Short: "Gracefully replace EC2 instances for given ECS cluster",
 	Long:  ``,
 	Run: func(cmd *cobra.Command, args []string) {
-
 		initAwsSess()
 
 		asgName := lib.GetAsgNameForEcsCluster(AwsSess, cluster)
 		if asgName == "" {
-			fmt.Println("Unable to find ASG name for ECS cluster ", cluster)
-			os.Exit(1)
+			log.Fatalln("Unable to find ASG name for ECS cluster ", cluster)
```

Comment:
*ln() should add a space between each parameter if I recall correctly. Can be changed multiple places.

```suggestion
			log.Fatalln("Unable to find ASG name for ECS cluster", cluster)
```